src/overlays/gamestates/ovl_trademark/m_trademark.c

```diff
@@ -0,0 +1,585 @@
+#include "m_trademark.h"
+
+#include "audio.h"
+#include "m_actor.h"
+#include "m_common_data.h"
+#include "m_event.h"
+#include "m_field_info.h"
+#include "m_flashrom.h"
+#include "m_mail.h"
+#include "m_npc.h"
+#include "m_scene_table.h"
+#include "m_time.h"
+#include "m_rcp.h"
+#include "m_view.h"
+#include "sys_math.h"
+#include "sys_matrix.h"
+#include "libc64/qrand.h"
+#include "m_std_dma.h"
+#include "67E840.h"
+#include "69CB30.h"
+#include "6B3DC0.h"
+#include "6B8F20.h"
+#include "6EC9E0.h"
+#include "segment_symbols.h"
+#include "macros.h"
+
+#include "overlays/gamestates/ovl_play/m_play.h"
+
+typedef enum TrademarkState {
+    /* 0 */ TRADEMARK_STATE_0,
+    /* 1 */ TRADEMARK_STATE_1,
+    /* 2 */ TRADEMARK_STATE_2,
+    /* 3 */ TRADEMARK_STATE_3,
+    /* 4 */ TRADEMARK_STATE_4,
+    /* 5 */ TRADEMARK_STATE_5,
+} TrademarkState;
+
+DoorData D_80805B20_jp = { 0x21, 4, 0, 0, { 0x884, 0xC8, 0x338 }, 0, 3, { 0, 0, 0 } };
```

Comment:
```suggestion
DoorData demo_1_door_data = { SCENE_TITLE_DEMO, 4, 0, 0, { 0x884, 0xC8, 0x338 }, 0, 3, { 0, 0, 0 } };
```

Reply:
For whatever reason I thought we didn't had the original names of those symbols. Nice